packages/core-mobile/app/components/SafeAreaViews.tsx

```diff
@@ -0,0 +1,31 @@
+import { View, type SxProp } from '@avalabs/k2-mobile'
+import React, { type FC, type PropsWithChildren } from 'react'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+
+type Props = {
+  sx?: SxProp
+  noPaddingTop?: boolean
+}
+
```

Comment:
can you add some comments in this file to briefly explain when to use SafeVertical vs SafeLower?

Reply:
I added some comments. Let me know if these aren't clear enough.